### PR TITLE
Add badges for build/doc and validation

### PR DESCRIPTION
**Pull request by ravwojdyla**
Re #264

### CODE REVIEW

**Review by jeromekelleher — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # sgkit
+[![Build status](https://github.com/pystatgen/sgkit/workflows/Check%20code%20standards%20and%20run%20tests/badge.svg?branch=master)](https://github.com/pystatgen/sgkit/actions?query=workflow%3A%22Check+code+standards+and+run+tests%22+branch%3Amaster)
+[![Documentation status](https://github.com/pystatgen/sgkit/workflows/Publish%20documentation/badge.svg?branch=master)](https://pystatgen.github.io/sgkit/)
```
Comment (tomwhite) [inline README.md:3]: Nit: might be nice to make these a bit shorter. E.g. "Build" for the first one, "Docs" for the second?

Comment (ravwojdyla) [reply README.md]: afaiu this 70e9a1b is required to make that happen

**Review by tomwhite — APPROVED** [review]

**Comment by ravwojdyla** [comment]
@tomwhite this PR changes github workflows so you need to merge it manually

**Pull request merged**